packages/extension-chakra-storefront/src/hooks/use-datacloud.test.js

```diff
@@ -99,7 +94,12 @@ MockComponent.propTypes = {
     args: PropTypes.array
 }
 
-describe.skip('useDataCloud', function () {
+describe('useDataCloud', function () {
+    beforeAll(() => {
+        Cookies.get.mockImplementation((key) => {
+            return key === 'sid' ? 'mocked-sessionId' : undefined
+        })
```

Comment:
Mocking js-cookie in this way does not run into out-of-memory error that I saw previously. Credit goes to @bendvc.